client/reader/stream/index.jsx

```diff
@@ -438,6 +503,22 @@ class ReaderStream extends React.Component {
 				/>;
 		}
 
+		if ( postKey.isCombination ) {
+			const handleClick = post => showSelectedPost( {
```

Comment:
@blowery, lets handle that in a separate PR and keep this focused on the new component